WKWebView.ios.js

```diff
@@ -448,7 +448,7 @@ class WKWebView extends React.Component {
     );
   };
 
-  evaluateJavaScript = (js) => {
+  injectJavaScript = (js) => {
     return WKWebViewManager.evaluateJavaScript(this.getWebViewHandle(), js);
```

Comment:
shouldn't `evaluateJavaScript` be replaced by `injectJavaScript` here?

Reply:
No, it's a method from native code. I've just changed the bridge.